project/CodeCoverageSettings.scala

```diff
@@ -0,0 +1,21 @@
+import sbt.Setting
+import scoverage.ScoverageKeys.*
+
+object CodeCoverageSettings {
+  private val excludedPackages: Seq[String] = Seq(
+    "<empty>",
+    ".*definition.*",
+    "prod.*",
+    "live.*",
+    "testOnlyDoNotUseInAppConf.*",
+    "app.*",
+    "uk.gov.hmrc.BuildInfo"
+  )
+
+  val settings: Seq[Setting[?]] = Seq(
+    coverageExcludedFiles := "<empty>;.*components.*;.*Routes.*;",
```

Comment:
the excluded packages are being set above. Just use the excludedPackages val here